docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1651 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #67.                    D = core._mass(Q, T[start: (stop+m-1)], QT, M_T[i], Σ_T[i], M_T[start:stop], Σ_T[start:stop])
</pre><p>This should be <code>stop - m + 1</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Could you please check it out again?

Because, I think we were correct and you did not make a mistake. So, for instance, let's say `cand_idx_chunks` is `[10,11,15]` . That means I need to find the distance of three subsequences (with **START** index `10`, `11`, and `15`) to `Q`. The chunks will be `[10,12]` and `[15,16]`. So, for the first chunk, we should create a time series that contains subsequences starting at `10` and `11`. So, the time series will be `T[start: (stop + m - 1)]=T[10: (12 + m - 1)]`.

Reply:
<p>Please ignore me :D</p>